benchmarks/run_all.sh

```diff
@@ -11,6 +11,7 @@ for file in \
   `dirname "$0"`/profiler_allocation.rb \
   `dirname "$0"`/profiler_gc.rb \
   `dirname "$0"`/profiler_hold_resume_interruptions.rb \
+  `dirname "$0"`/profiler_http_transport.rb \
```

Comment:
This was temporarily removed in https://github.com/DataDog/dd-trace-rb/pull/4569 and I'm re-adding it here. TL;DR whenever a benchmark uses an API that changes our current setup breaks so the benchmark needs to be disabled and then re-enabled.